bindings/python/tests/test_pt_comparison.py

```diff
@@ -29,6 +29,38 @@ def test_zero_sized(self):
         reloaded = load_file(local)
         self.assertTrue(torch.equal(data["test"], reloaded["test"]))
 
+    def test_multiple_zero_sized(self):
+        data = {
+            "test": torch.zeros((2, 0), dtype=torch.float),
+            "test2": torch.zeros((2, 0), dtype=torch.float),
+        }
+        local = "./tests/data/out_safe_pt_mmap_small3.safetensors"
+        save_file(data, local)
+        reloaded = load_file(local)
+        self.assertTrue(torch.equal(data["test"], reloaded["test"]))
+        self.assertTrue(torch.equal(data["test2"], reloaded["test2"]))
+
+    def test_disjoint_tensors_shared_storage(self):
+        A = torch.zeros((10, 10))
```

Comment:
Nit, not fan of capitalized names for variables. Not fan of one-letter names either ;-)

Reply:
Do you have a better suggestion ?

Reply:
`tensor`, `array`, `matrix`